lib/elliptic/curve/mont.js

```diff
@@ -147,6 +156,10 @@ Point.prototype.mulAdd = function mulAdd() {
   throw new Error('Not supported on Montgomery curve');
 };
 
+Point.prototype.eq = function eq(other) {
```

Comment:
I think I added an `eq` method to the edwards curve, and short already had one. Maybe it would be good to declare an  abstract method using the preexisting convention, similar to:

```javascript
BaseCurve.prototype.point = function point() {
  throw new Error('Not implemented');
};
```
eg.

```javascript
BasePoint.prototype.eq = function eq(other) {
  throw new Error('Not implemented');
};
```

Doesn't seem there's much in common